frontend/test/unit/setup.js

```diff
@@ -19,7 +19,13 @@ Vue.use(VueI18n)
  */
 config.stubs["nuxt-link"] = Vue.component("NuxtLink", {
   props: ["to"],
-  template: '<a :href="to" v-on="$listeners"><slot /></a>',
+  methods: {
+    handleClick() {
+      this.$emit("mousedown")
+      this.$emit("click", new MouseEvent("click"))
+    },
+  },
+  template: '<a :href="to" v-on="$listeners" @click="handleClick"><slot /></a>',
```

Comment:
Can a note be added that this adaptation is necessary to mimic NuxtLink's behaviour (and fix a bug)? If we can remove this after the Nuxt 3 upgrade, noting that as well would be nice.